app/assets/stylesheets/sufia/_file-listing.scss

```diff
@@ -170,6 +178,10 @@ input#check_all {
   margin-bottom: 10px;
 }
 
+#appliedParams.constraints-container {
```

Comment:
And the ID selector is like the other one, didn't want to bork anything else that used the class